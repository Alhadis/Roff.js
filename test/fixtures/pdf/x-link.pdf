%PDF-1.4
%‚„œ”
3 0 obj << /Annots [10 0 R  11 0 R  12 0 R  13 0 R  14 0 R  ]
/Contents [4 0 R  ]
/Group << /CS /DeviceRGB
/S /Transparency
>>
/Parent 2 0 R
/Type /Page
>>
endobj
4 0 obj << /Length 10052
>>
stream
q 1 0 0 1 0 0 cm
1 J
1 j
% x font 36 TB
% f36
% s12000
q BT
1 0 0 1 0 792 Tm
/F36 10 Tf
0 Tc
% V18000
/F36 12 Tf
%% H72000
% md
1 0 0 1 72 774 Tm
0 Tc
0 g
% DFd
ET Q
0 G
0 g
% tLine
q BT
1 0 0 1 72 774 Tm
/F36 12 Tf
0 Tc
% == 'Line'=23.34 'xpos=95.34
% --- wht=3000, pend=0, nomv=0
% C\-
% == '≠'=6.84 'xpos=102.18
% --- wht=3000, pend=0, nomv=0
% h6840
% tbr
% == 'br'=12 'xpos=114.18
% --- wht=3000, pend=6840, nomv=6840
% H113964
0 Tw (Line≠br) Tj
41.964 0 Td
% teak
% == 'eak'=18 'xpos=131.964
% --- wht=3000, pend=0, nomv=0
% wh3000
% ttest
% == 'test'=17.988 'xpos=152.952
% --- wht=3000, pend=3000, nomv=0
% n18000 0
0.000 Tw [(eak test)] TJ
% x font 5 TR
% f5
% V54000
/F5 12 Tf
%% H72000
% tMethion
1 0 0 1 72 738 Tm
0 Tc
% == 'Methion'=40.668 'xpos=112.668
% --- wht=3000, pend=0, nomv=0
% H112488
0 Tw (Methion) Tj
40.488 0 Td
% tylthreon
% == 'ylthreon'=39.996 'xpos=152.484
% --- wht=3000, pend=0, nomv=0
% H152304
0 Tw (ylthreon) Tj
39.816 0 Td
% tylthreon
% == 'ylthreon'=39.996 'xpos=192.3
% --- wht=3000, pend=0, nomv=0
% H192120
0 Tw (ylthreon) Tj
39.816 0 Td
% tylglutamin
% == 'ylglutamin'=52.008 'xpos=244.128
% --- wht=3000, pend=0, nomv=0
% H243948
0 Tw (ylglutamin) Tj
51.828 0 Td
% tylar
% == 'ylar'=18.66 'xpos=262.608
% --- wht=3000, pend=0, nomv=0
% H262392
0 Tw (ylar) Tj
18.444 0 Td
% tgin
% == 'gin'=15.336 'xpos=277.728
% --- wht=3000, pend=0, nomv=0
% H277548
0 Tw (gin) Tj
15.156 0 Td
% tyltyrosylgluta
% == 'yltyrosylgluta'=66.672 'xpos=344.22
% --- wht=3000, pend=0, nomv=0
% wh14520
% V54000
0 Tw [(yltyrosylgluta) -1210 ] TJ
%% H358740
% mr 22937 0 39321
1 0 0 1 358.74 738 Tm
0 Tc
0.349996185244526 0 0.6 rg
% x X pdf: markstart
%% + 8196 -2616 2000
%% + /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
%% + /Border [0 0 4 [40 10]]
%% + /Color  [0.35 0.00 0.60]
%% tmylserylleuc
% tmylserylleuc
% == 'mylserylleuc'=61.992 'xpos=420.732
% --- wht=3000, pend=0, nomv=0
% H420552
0 Tw (mylserylleuc) Tj
61.812 0 Td
% tylphen
% == 'ylphen'=32.664 'xpos=453.216
% --- wht=3000, pend=0, nomv=0
% H453036
0 Tw (ylphen) Tj
32.484 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=482.364
% --- wht=3000, pend=0, nomv=0
% H482184
0 Tw (ylalan) Tj
29.148 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=511.512
% --- wht=3000, pend=0, nomv=0
% H511332
0 Tw (ylalan) Tj
29.148 0 Td
% tylglu
% == 'ylglu'=24.672 'xpos=536.004
% --- wht=3000, pend=0, nomv=0
% Chy
% == '-'=3.996 'xpos=540
% --- wht=3000, pend=0, nomv=0
% h3996
% n18000 0
0 Tw (ylglu-) Tj
% V72000
%% H72000
% ttamin
1 0 0 1 72 720 Tm
0 Tc
% == 'tamin'=27.336 'xpos=99.336
% --- wht=3000, pend=0, nomv=0
% H99156
0 Tw (tamin) Tj
27.156 0 Td
% tylleuc
% == 'ylleuc'=29.328 'xpos=128.484
% --- wht=3000, pend=0, nomv=0
% H128304
0 Tw (ylleuc) Tj
29.148 0 Td
% tyllysylglutamylar
% == 'yllysylglutamylar'=84.672 'xpos=212.976
% --- wht=3000, pend=0, nomv=0
% H212760
0 Tw (yllysylglutamylar) Tj
84.456 0 Td
% tgin
% == 'gin'=15.336 'xpos=228.096
% --- wht=3000, pend=0, nomv=0
% H227916
0 Tw (gin) Tj
15.156 0 Td
% tyllysylglutamylglyc
% == 'yllysylglutamylglyc'=96.012 'xpos=323.928
% --- wht=3000, pend=0, nomv=0
% H323748
0 Tw (yllysylglutamylglyc) Tj
95.832 0 Td
% x X pdf: marksuspend
%% tylalpooan
% tylalpooan
% == 'ylalpooan'=47.328 'xpos=371.076
% --- wht=3000, pend=0, nomv=0
% H370896
0 Tw (ylalpooan) Tj
47.148 0 Td
% tylphen
% == 'ylphen'=32.664 'xpos=403.56
% --- wht=3000, pend=0, nomv=0
% H403380
0 Tw (ylphen) Tj
32.484 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=432.708
% --- wht=3000, pend=0, nomv=0
% H432528
0 Tw (ylalan) Tj
29.148 0 Td
% tylv
% == 'ylv'=15.336 'xpos=447.864
% --- wht=3000, pend=0, nomv=0
% H447564
0 Tw (ylv) Tj
15.036 0 Td
% talylprolylphen
% == 'alylprolylphen'=69.996 'xpos=517.56
% --- wht=3000, pend=0, nomv=0
% H517380
0 Tw (alylprolylphen) Tj
69.816 0 Td
% ty
% == 'y'=6 'xpos=523.38
% --- wht=3000, pend=0, nomv=0
% Chy
% == '-'=3.996 'xpos=527.376
% --- wht=3000, pend=0, nomv=0
% h3996
% n18000 0
0 Tw (y-) Tj
% x X pdf: markrestart
%% V90000
% V90000
%% H72000
% tlalan
1 0 0 1 72 702 Tm
0 Tc
% == 'lalan'=23.328 'xpos=95.328
% --- wht=3000, pend=0, nomv=0
% H95148
0 Tw (lalan) Tj
23.148 0 Td
% tylv
% == 'ylv'=15.336 'xpos=110.484
% --- wht=3000, pend=0, nomv=0
% H110184
0 Tw (ylv) Tj
15.036 0 Td
% talylthreon
% == 'alylthreon'=48.66 'xpos=158.844
% --- wht=3000, pend=0, nomv=0
% H158664
0 Tw (alylthreon) Tj
48.480 0 Td
% tylleuc
% == 'ylleuc'=29.328 'xpos=187.992
% --- wht=3000, pend=0, nomv=0
% H187812
0 Tw (ylleuc) Tj
29.148 0 Td
% tylgl
% == 'ylgl'=18.672 'xpos=206.484
% --- wht=3000, pend=0, nomv=0
% V90000
0 Tw (ylgl) Tj
%% H206484
% x X pdf: markend
%% wh13092
% wh13092
% md
1 0 0 1 206.484 702 Tm
0 Tc
0 g
% tyc
% == 'yc'=11.328 'xpos=230.904
% --- wht=3000, pend=13092, nomv=0
% H230724
10.092 Tw [ -1091 (yc)] TJ
24.240 0 Td
% tylaspartylprolylglyc
% == 'ylaspartylprolylglyc'=96.66 'xpos=327.384
% --- wht=3000, pend=0, nomv=0
% H327204
0 Tw (ylaspartylprolylglyc) Tj
96.480 0 Td
% tylisoleuc
% == 'ylisoleuc'=43.332 'xpos=370.536
% --- wht=3000, pend=0, nomv=0
% H370356
0 Tw (ylisoleuc) Tj
43.152 0 Td
% tylglutamylglutamin
% == 'ylglutamylglutamin'=94.68 'xpos=465.036
% --- wht=3000, pend=0, nomv=0
% H464856
0 Tw (ylglutamylglutamin) Tj
94.500 0 Td
% tylserylleuc
% == 'ylserylleuc'=52.656 'xpos=517.512
% --- wht=3000, pend=0, nomv=0
% H517332
0 Tw (ylserylleuc) Tj
52.476 0 Td
% tylly
% == 'ylly'=18.672 'xpos=536.004
% --- wht=3000, pend=0, nomv=0
% Chy
% == '-'=3.996 'xpos=540
% --- wht=3000, pend=0, nomv=0
% h3996
% n18000 0
0 Tw (ylly-) Tj
% V108000
%% H72000
% tsylisoleuc
1 0 0 1 72 684 Tm
0 Tc
% == 'sylisoleuc'=48 'xpos=120
% --- wht=3000, pend=0, nomv=0
% H119820
0 Tw (sylisoleuc) Tj
47.820 0 Td
% tylaspartylthreon
% == 'ylaspartylthreon'=77.988 'xpos=197.808
% --- wht=3000, pend=0, nomv=0
% H197628
0 Tw (ylaspartylthreon) Tj
77.808 0 Td
% tylleuc
% == 'ylleuc'=29.328 'xpos=226.956
% --- wht=3000, pend=0, nomv=0
% H226776
0 Tw (ylleuc) Tj
29.148 0 Td
% tylisoleu
% == 'ylisoleu'=38.004 'xpos=264.78
% --- wht=3000, pend=0, nomv=0
% wh14328
% tc
% == 'c'=5.328 'xpos=284.436
% --- wht=3000, pend=14328, nomv=0
% H284256
11.328 Tw [(ylisoleu c)] TJ
57.480 0 Td
% tylglutamylalan
% == 'ylglutamylalan'=72 'xpos=356.256
% --- wht=3000, pend=0, nomv=0
% H356076
0 Tw (ylglutamylalan) Tj
71.820 0 Td
% tylglyc
% == 'ylglyc'=30 'xpos=386.076
% --- wht=3000, pend=0, nomv=0
% H385896
0 Tw (ylglyc) Tj
29.820 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=415.224
% --- wht=3000, pend=0, nomv=0
% H415044
0 Tw (ylalan) Tj
29.148 0 Td
% tylaspartylalan
% == 'ylaspartylalan'=67.32 'xpos=482.364
% --- wht=3000, pend=0, nomv=0
% H482184
0 Tw (ylaspartylalan) Tj
67.140 0 Td
% tylleuc
% == 'ylleuc'=29.328 'xpos=511.512
% --- wht=3000, pend=0, nomv=0
% H511332
0 Tw (ylleuc) Tj
29.148 0 Td
% tylglu
% == 'ylglu'=24.672 'xpos=536.004
% --- wht=3000, pend=0, nomv=0
% Chy
% == '-'=3.996 'xpos=540
% --- wht=3000, pend=0, nomv=0
% h3996
% n18000 0
0 Tw (ylglu-) Tj
% V126000
%% H72000
% ttamylleuc
1 0 0 1 72 666 Tm
0 Tc
% == 'tamylleuc'=47.328 'xpos=119.328
% --- wht=3000, pend=0, nomv=0
% H119148
0 Tw (tamylleuc) Tj
47.148 0 Td
% tylglyc
% == 'ylglyc'=30 'xpos=149.148
% --- wht=3000, pend=0, nomv=0
% H148968
0 Tw (ylglyc) Tj
29.820 0 Td
% ty
% == 'y'=6 'xpos=154.968
% --- wht=3000, pend=0, nomv=0
% wh11888
% V126000
0 Tw [(y) -990.666666666667 ] TJ
%% H166856
% mr 22937 0 39321
1 0 0 1 166.856 666 Tm
0 Tc
0.349996185244526 0 0.6 rg
% x X pdf: markstart
%% + 8196 -2616 2000
%% + /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
%% + /Border [0 0 1]
%% + /Color  [0.35 0.00 0.60]
%% tlisoleuc
% tlisoleuc
% == 'lisoleuc'=37.332 'xpos=204.188
% --- wht=3000, pend=0, nomv=0
% H204008
0 Tw (lisoleuc) Tj
37.152 0 Td
% tylprolylphen
% == 'ylprolylphen'=61.332 'xpos=265.34
% --- wht=3000, pend=0, nomv=0
% H265160
0 Tw (ylprolylphen) Tj
61.152 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=294.488
% --- wht=3000, pend=0, nomv=0
% H294308
0 Tw (ylalan) Tj
29.148 0 Td
% tylseryla
% == 'ylseryla'=37.992 'xpos=332.3
% --- wht=3000, pend=0, nomv=0
% V126000
0 Tw (ylseryla) Tj
%% H332300
% x X pdf: markend
%% wh11888
% wh11888
% md
1 0 0 1 332.3 666 Tm
0 Tc
0 g
% tc
% == 'c'=5.328 'xpos=349.516
% --- wht=3000, pend=11888, nomv=0
% H349336
8.888 Tw [ -990.666666666667 (c)] TJ
17.036 0 Td
% tylglutamylalan
% == 'ylglutamylalan'=72 'xpos=421.336
% --- wht=3000, pend=0, nomv=0
% H421156
0 Tw (ylglutamylalan) Tj
71.820 0 Td
% tylglyc
% == 'ylglyc'=30 'xpos=451.156
% --- wht=3000, pend=0, nomv=0
% H450976
0 Tw (ylglyc) Tj
29.820 0 Td
% tylalan
% == 'ylalan'=29.328 'xpos=480.304
% --- wht=3000, pend=0, nomv=0
% H480124
0 Tw (ylalan) Tj
29.148 0 Td
% tylas
% == 'ylas'=19.332 'xpos=499.456
% --- wht=3000, pend=0, nomv=0
% wh11888
% V126000
0 Tw [(ylas) -990.666666666667 ] TJ
%% H511344
% mr 22937 0 39321
1 0 0 1 511.344 666 Tm
0 Tc
0.349996185244526 0 0.6 rg
% x X pdf: markstart
%% + 8196 -2616 2000
%% + /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
%% + /Border [0 0 1]
%% + /Color  [0.35 0.00 0.60]
%% tparty
% tparty
% == 'party'=24.66 'xpos=536.004
% --- wht=3000, pend=0, nomv=0
% Chy
% == '-'=3.996 'xpos=540
% --- wht=3000, pend=0, nomv=0
% h3996
% n18000 0
0 Tw (party-) Tj
% V144000
%% H72000
% tlalan
1 0 0 1 72 648 Tm
0 Tc
% == 'lalan'=23.328 'xpos=95.328
% --- wht=3000, pend=0, nomv=0
% H95148
0 Tw (lalan) Tj
23.148 0 Td
% tylleuc
% == 'ylleuc'=29.328 'xpos=124.476
% --- wht=3000, pend=0, nomv=0
% H124296
0 Tw (ylleuc) Tj
29.148 0 Td
% tyl
% == 'yl'=9.336 'xpos=133.632
% --- wht=3000, pend=0, nomv=0
% V144000
0 Tw (yl) Tj
%% H133632
% x X pdf: markend
%% wh3000
% wh3000
% md
1 0 0 1 133.632 648 Tm
0 Tc
0 g
% tglutamylleuc
% == 'glutamylleuc'=62.664 'xpos=199.296
% --- wht=3000, pend=3000, nomv=0
% H199116
0.000 Tw [ -250 (glutamylleuc)] TJ
65.484 0 Td
% tylglyc
% == 'ylglyc'=30 'xpos=229.116
% --- wht=3000, pend=0, nomv=0
% H228936
0 Tw (ylglyc) Tj
29.820 0 Td
% ty
% == 'y'=6 'xpos=234.936
% --- wht=3000, pend=0, nomv=0
% n18000 0
0 Tw (y) Tj
% x trailer
% V792000
%% x stop
% x stop
1 0 0 1 234.936 0 Tm
0 Tc
ET Q
Q
endstream
endobj
1 0 obj << /Pages 2 0 R 
/Type /Catalog
>>
endobj
15 0 obj << /CreationDate (D:20170706194731+10'00')
/Creator (groff version 1.22.3)
/ModDate (D:20170706194731+10'00')
/Producer (gropdf version 1.22.3)
>>
endobj
5 0 obj << /BaseFont /Times-Bold
/Encoding 6 0 R 
/Subtype /Type1
/ToUnicode 7 0 R 
/Type /Font
>>
endobj
6 0 obj << /Differences [0 /asciicircum /asciitilde /Scaron /Zcaron /scaron /zcaron /Ydieresis /trademark /quotesingle /Euro /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /space /exclam /quotedbl /numbersign /dollar /percent /ampersand /quoteright /parenleft /parenright /asterisk /plus /comma /hyphen /period /slash /zero /one /two /three /four /five /six /seven /eight /nine /colon /semicolon /less /equal /greater /question /at /A /B /C /D /E /F /G /H /I /J /K /L /M /N /O /P /Q /R /S /T /U /V /W /X /Y /Z /bracketleft /backslash /bracketright /circumflex /underscore /quoteleft /a /b /c /d /e /f /g /h /i /j /k /l /m /n /o /p /q /r /s /t /u /v /w /x /y /z /braceleft /bar /braceright /tilde /.notdef /quotesinglbase /guillemotleft /guillemotright /bullet /florin /fraction /perthousand /dagger /daggerdbl /endash /emdash /.notdef /fi /fl /.notdef /.notdef /dotlessi /.notdef /grave /hungarumlaut /dotaccent /breve /caron /ring /ogonek /quotedblleft /quotedblright /oe /lslash /quotedblbase /OE /Lslash /.notdef /exclamdown /cent /sterling /currency /yen /brokenbar /section /dieresis /copyright /ordfeminine /guilsinglleft /logicalnot /minus /registered /macron /degree /plusminus /twosuperior /threesuperior /acute /mu /paragraph /periodcentered /cedilla /onesuperior /ordmasculine /guilsinglright /onequarter /onehalf /threequarters /questiondown /Agrave /Aacute /Acircumflex /Atilde /Adieresis /Aring /AE /Ccedilla /Egrave /Eacute /Ecircumflex /Edieresis /Igrave /Iacute /Icircumflex /Idieresis /Eth /Ntilde /Ograve /Oacute /Ocircumflex /Otilde /Odieresis /multiply /Oslash /Ugrave /Uacute /Ucircumflex /Udieresis /Yacute /Thorn /germandbls /agrave /aacute /acircumflex /atilde /adieresis /aring /ae /ccedilla /egrave /eacute /ecircumflex /edieresis /igrave /iacute /icircumflex /idieresis /eth /ntilde /ograve /oacute /ocircumflex /otilde /odieresis /divide /oslash /ugrave /uacute /ucircumflex /udieresis /yacute /thorn /ydieresis /ellipsis ]
/Type /Encoding
>>
endobj
7 0 obj << /Length 426
>>
stream
/CIDInit /ProcSet findresource begin
12 dict begin
begincmap
/CIDSystemInfo
<< /Registry (Adobe)
/Ordering (UCS)
/Supplement 0
>> def
/CMapName /Adobe-Identity-UCS def
/CMapType 2 def
1 begincodespacerange
<0000> <FFFF>
endcodespacerange
2 beginbfrange
<008b> <008f> [<00660066> <00660069> <0066006c> <006600660069> <00660066006C>]
<00ad> <00ad> <002d>
endbfrange
endcmap
CMapName currentdict /CMap defineresource pop
end
end
endstream
endobj
8 0 obj << /BaseFont /Times-Roman
/Encoding 9 0 R 
/Subtype /Type1
/ToUnicode 7 0 R 
/Type /Font
>>
endobj
9 0 obj << /Differences [0 /asciicircum /asciitilde /Scaron /Zcaron /scaron /zcaron /Ydieresis /trademark /quotesingle /Euro /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /space /exclam /quotedbl /numbersign /dollar /percent /ampersand /quoteright /parenleft /parenright /asterisk /plus /comma /hyphen /period /slash /zero /one /two /three /four /five /six /seven /eight /nine /colon /semicolon /less /equal /greater /question /at /A /B /C /D /E /F /G /H /I /J /K /L /M /N /O /P /Q /R /S /T /U /V /W /X /Y /Z /bracketleft /backslash /bracketright /circumflex /underscore /quoteleft /a /b /c /d /e /f /g /h /i /j /k /l /m /n /o /p /q /r /s /t /u /v /w /x /y /z /braceleft /bar /braceright /tilde /.notdef /quotesinglbase /guillemotleft /guillemotright /bullet /florin /fraction /perthousand /dagger /daggerdbl /endash /emdash /.notdef /fi /fl /.notdef /.notdef /dotlessi /.notdef /grave /hungarumlaut /dotaccent /breve /caron /ring /ogonek /quotedblleft /quotedblright /oe /lslash /quotedblbase /OE /Lslash /.notdef /exclamdown /cent /sterling /currency /yen /brokenbar /section /dieresis /copyright /ordfeminine /guilsinglleft /logicalnot /minus /registered /macron /degree /plusminus /twosuperior /threesuperior /acute /mu /paragraph /periodcentered /cedilla /onesuperior /ordmasculine /guilsinglright /onequarter /onehalf /threequarters /questiondown /Agrave /Aacute /Acircumflex /Atilde /Adieresis /Aring /AE /Ccedilla /Egrave /Eacute /Ecircumflex /Edieresis /Igrave /Iacute /Icircumflex /Idieresis /Eth /Ntilde /Ograve /Oacute /Ocircumflex /Otilde /Odieresis /multiply /Oslash /Ugrave /Uacute /Ucircumflex /Udieresis /Yacute /Thorn /germandbls /agrave /aacute /acircumflex /atilde /adieresis /aring /ae /ccedilla /egrave /eacute /ecircumflex /edieresis /igrave /iacute /icircumflex /idieresis /eth /ntilde /ograve /oacute /ocircumflex /otilde /odieresis /divide /oslash /ugrave /uacute /ucircumflex /udieresis /yacute /thorn /ydieresis /ellipsis ]
/Type /Encoding
>>
endobj
10 0 obj << /A << /S /URI
/URI (http://go.to/)
>>
/Border [0 0 4 [40 10 ] ]
/C [0.35 0.00 0.60 ]
/Rect [356.74 733.384 542 748.196 ]
/Subtype /Link
/Type /Annot
>>
endobj
11 0 obj << /A << /S /URI
/URI (http://go.to/)
>>
/Border [0 0 4 [40 10 ] ]
/C [0.35 0.00 0.60 ]
/Rect [72 697.384 208.484 712.196 ]
/Subtype /Link
/Type /Annot
>>
endobj
12 0 obj << /A << /S /URI
/URI (http://go.to/)
>>
/Border [0 0 1 ]
/C [0.35 0.00 0.60 ]
/Rect [164.856 661.384 334.3 676.196 ]
/Subtype /Link
/Type /Annot
>>
endobj
13 0 obj << /A << /S /URI
/URI (http://go.to/)
>>
/Border [0 0 1 ]
/C [0.35 0.00 0.60 ]
/Rect [509.344 661.384 542 676.196 ]
/Subtype /Link
/Type /Annot
>>
endobj
14 0 obj << /A << /S /URI
/URI (http://go.to/)
>>
/Border [0 0 1 ]
/C [0.35 0.00 0.60 ]
/Rect [72 643.384 135.632 658.196 ]
/Subtype /Link
/Type /Annot
>>
endobj
2 0 obj << /Count 1
/Kids [3 0 R  ]
/MediaBox [0 0 612 792 ]
/Resources << /Font << /F36 5 0 R 
/F5 8 0 R 
>>
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI ]
>>
/Rotate 0
/Type /Pages
>>
endobj
xref
0 16
0000000000 65535 f 
0000010282 00000 n 
0000016276 00000 n 
0000000015 00000 n 
0000000178 00000 n 
0000010495 00000 n 
0000010601 00000 n 
0000012731 00000 n 
0000013207 00000 n 
0000013314 00000 n 
0000015444 00000 n 
0000015615 00000 n 
0000015786 00000 n 
0000015951 00000 n 
0000016114 00000 n 
0000010332 00000 n 
trailer
<<
/Info 15 0 R 
/Root 1 0 R
/Size 16
>>
startxref
16469
%%EOF
