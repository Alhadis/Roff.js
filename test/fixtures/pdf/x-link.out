x T pdf
x res 72000 1 1
x init
x F fixtures/x-link.pic
p1
x font 36 TB
f36
s12000
V18000
H72000
md
DFd
tLine
C\-
h6840
tbr
H113964
teak
wh3000
ttest
n18000 0
x font 5 TR
f5
V54000
H72000
tMethion
H112488
tylthreon
H152304
tylthreon
H192120
tylglutamin
H243948
tylar
H262392
tgin
H277548
tyltyrosylgluta
wh14520
V54000
H358740
mr 22937 0 39321
x X pdf: markstart
+ 8196 -2616 2000
+ /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
+ /Border [0 0 4 [40 10]]
+ /Color  [0.35 0.00 0.60]
tmylserylleuc
H420552
tylphen
H453036
tylalan
H482184
tylalan
H511332
tylglu
Chy
h3996
n18000 0
V72000
H72000
ttamin
H99156
tylleuc
H128304
tyllysylglutamylar
H212760
tgin
H227916
tyllysylglutamylglyc
H323748
x X pdf: marksuspend
tylalpooan
H370896
tylphen
H403380
tylalan
H432528
tylv
H447564
talylprolylphen
H517380
ty
Chy
h3996
n18000 0
x X pdf: markrestart
V90000
H72000
tlalan
H95148
tylv
H110184
talylthreon
H158664
tylleuc
H187812
tylgl
V90000
H206484
x X pdf: markend
wh13092
md
tyc
H230724
tylaspartylprolylglyc
H327204
tylisoleuc
H370356
tylglutamylglutamin
H464856
tylserylleuc
H517332
tylly
Chy
h3996
n18000 0
V108000
H72000
tsylisoleuc
H119820
tylaspartylthreon
H197628
tylleuc
H226776
tylisoleu
wh14328
tc
H284256
tylglutamylalan
H356076
tylglyc
H385896
tylalan
H415044
tylaspartylalan
H482184
tylleuc
H511332
tylglu
Chy
h3996
n18000 0
V126000
H72000
ttamylleuc
H119148
tylglyc
H148968
ty
wh11888
V126000
H166856
mr 22937 0 39321
x X pdf: markstart
+ 8196 -2616 2000
+ /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
+ /Border [0 0 1]
+ /Color  [0.35 0.00 0.60]
tlisoleuc
H204008
tylprolylphen
H265160
tylalan
H294308
tylseryla
V126000
H332300
x X pdf: markend
wh11888
md
tc
H349336
tylglutamylalan
H421156
tylglyc
H450976
tylalan
H480124
tylas
wh11888
V126000
H511344
mr 22937 0 39321
x X pdf: markstart
+ 8196 -2616 2000
+ /Subtype /Link /Action << /Subtype /URI /URI (http://go.to/) >>
+ /Border [0 0 1]
+ /Color  [0.35 0.00 0.60]
tparty
Chy
h3996
n18000 0
V144000
H72000
tlalan
H95148
tylleuc
H124296
tyl
V144000
H133632
x X pdf: markend
wh3000
md
tglutamylleuc
H199116
tylglyc
H228936
ty
n18000 0
x trailer
V792000
x stop
